src/runtime/logging.cc

```diff
@@ -222,33 +222,45 @@ TvmLogDebugSettings TvmLogDebugSettings::ParseSpec(const char* opt_spec) {
     return settings;
   }
   std::istringstream spec_stream(spec);
+  auto tell_pos = [&](const std::string& last_read) {
```

Comment:
While you're in there could you remove the '/' prefix from kSrcPrefix (and change the length). I keep getting caught by src/x/y/z.cc=1 not working.

Reply:
fixed